python/makeALPACAEvents.py

```diff
@@ -139,6 +144,59 @@ def runEvents(mres,gax,nev,Lmin,Lmax,startZ,endZ,SmearBeam):
     inputFile = ntupleWrite(ctau,fn,Lmin,Lmax,startZ,endZ,SmearBeam,mres,gax,old)
     os.chdir(old)
+    os.system("rm "+wdir+"/input_"+str(mres)+"_"+str(gax)+".DAT")
+    os.system("rm "+wdir+"/evrecs/evrec_"+str(mres)+"_"+str(gax)+".dat")
+    os.system("rm "+wdir+"/outputs/output_"+str(mres)+"_"+str(gax)+".dat")
     print('Ntuple is ready for the reading.')
     return inputFile
-#runEvents()
+ 
+def main():
+    from argparse import ArgumentParser
+    mres      = 1.0e-1
+    gax       = 1.0e-5
+    nev       = 100
+    Lmin      = 4.7505e0
+    Lmax      = 9.826511e0
+    startZ    = -7228.5000
+    endZ      = -7084.0000
+    SmearBeam = 1.0
+    print("ALPACA is running standalone")
+    parser = ArgumentParser()
+    parser.add_argument("-m", "--mres", dest="mres",  help="to set mass in GeV for ALP", required=False, default=None, type=float)
+    parser.add_argument("-g", "--gax", dest="gax",  help="to set photon coupling coefficient", required=False,default=None, type=float)
+    parser.add_argument("-Ls", "--Lmin", dest="Lmin",  help="to set starting length(m) of Decay Volume", required=False,default=None, type=float)
+    parser.add_argument("-Le", "--Lmax", dest="Lmax",  help="to set ending length(m) of Decay Volume", required=False,default=None, type=float)
+    parser.add_argument("-sZ", "--startZ", dest="startZ",  help="to set starting coordinate(cm) of Target", required=False,default=None, type=float)
+    parser.add_argument("-eZ", "--endZ", dest="endZ",  help="to set ending coordinate(cm) of Target", required=False,default=None, type=float)
+    parser.add_argument("-n", "--nev",dest="nev",  help="Number of events to generate", required=False,  default=None, type=int)
+    options = parser.parse_args()
+    if options.mres: 
+        mres=options.mres
+        print("mas is set to {} GeV".format(mres))
+    else: print("default mass is {} GeV".format(mres))
+    if options.gax:
+        gax=options.gax
+        print("photon coupling is set to {} GeV^-1".format(gax))
+    else: print("default photon coupling is {} GeV^-1".format(gax))
+    if options.Lmin:
+        Lmin=options.Lmin 
+        print("Lmin is set to {} m".format(Lmin))
+    else: print("default Lmin is {} m".format(Lmin))
+    if options.Lmax:
+        Lmax=options.Lmax 
+        print("Lmax is set to {} m".format(Lmax))
+    else: print("default Lmax is {} m".format(Lmax))
+    if options.startZ:
+        startZ=options.startZ 
+        print("startZ is set to {} cm".format(startZ))
+    else: print("default startZ is {} cm".format(startZ))
+    if options.endZ:
+        endZ=options.endZ
+        print("endZ is set to {} cm".format(endZ))
+    else: print("default endZ is {} cm".format(endZ))
+    if options.nev:
+        nev=options.nev  
+        print("nEvents is set to {}".format(nev))
+    else: print("default nEvents is {}".format(nev))
+    runEvents(mres,gax,nev,Lmin,Lmax,startZ,endZ,SmearBeam)   
+#main()
```

Comment:
Usually it's a good idea to do
```python
if __name__='__main__':
    main()
```
Which makes sure that `main()` is run when the script is called directly, but **not** if it is imported in e.g. `run_simScript.py`.

Reply:
Thank you for your hint.
Done.